api/helpers.js

```diff
@@ -22,9 +22,10 @@ const buildBirthParams = (attrs) => _.pickBy({
   forenames: attrs.forenames,
   dateofbirth: attrs.dob && toInternationalDateFormat(attrs.dob)
 });
-const buildAuditParams = (attrs) => ({
+const buildAuditParams = (attrs) => _.pickBy({
   from: toInternationalDateFormat(attrs.from),
-  to: toInternationalDateFormat(attrs.to)
+  to: toInternationalDateFormat(attrs.to),
+  user: attrs.user
 });
```

Comment:
Can we have an explicit _.identity rather than relying on the default? Just so that it's obvious what the intention is and that this is not a bug?